recipes/log4cxx/all/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.13)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if (NOT CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 17)
+endif()
```

Comment:
already in upstream CMakeLists:

https://github.com/apache/logging-log4cxx/blob/798106a705c0d4bf1fd4e59c14e061cca0d11663/CMakeLists.txt#L38-L42

By the way, does it compile with older standard? The comment in the link would suggest that C++17 is prefered but not required.

Reply:
I saw that log4cxx uses std::shared_mutex that is available since C++17. I've left "todo" for trying to reduce requirements by introducing dependency to boost::thread